pkg/queue/workers/metrics.go

```diff
@@ -1,174 +0,0 @@
-package workers
-
-import (
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
-)
-
-// largest bucket is 83 min
-var (
-	durationSBuckets  = []float64{1, 10, 50, 100, 200, 300, 500, 1000, 2000, 3000, 5000}
-	durationMsBuckets = []float64{10, 50, 100, 200, 300, 500, 1000, 2000, 3000, 5000}
-	queueMetricLabels = []string{"queue"}
-	taskMetricLabels  = []string{"queue", "type"}
-)
-
-// TaskQueueMetrics provides access to the prometheus metric objects for the task queue
-var TaskQueueMetrics = struct {
-	Labels             []string
-	TaskDuration       *prometheus.HistogramVec
-	TaskWaiting        *prometheus.HistogramVec
-	DequeueDuration    prometheus.Histogram
-	WorkerGauge        prometheus.Gauge
-	WorkerWaiting      prometheus.Counter
-	WorkerWorking      *prometheus.CounterVec
-	WorkerWorkingGauge *prometheus.GaugeVec
-	WorkerTask         *prometheus.CounterVec
-	WorkerErrors       *prometheus.CounterVec
```

Comment:
Which of the new metrics is the equivalent of this metric?  This one is important if we want to say "metadata sync has a lot of errors" because it is a CounterVec

Reply:
Just added.